packages/apollo/src/commands/service/__tests__/__snapshots__/check.test.ts.snap

```diff
@@ -148,7 +148,6 @@ exports[`service:check integration federated should report composition success c
 exports[`service:check integration federated should report composition success correctly compacts output in CI 1`] = `
 "Loading Apollo Project
 Found 0 graph composition errors for service accounts on graph engine
-Validated composed schema against tag master on graph engine
 Compared 1 schema change against 0 operations over the last 548 days
```

Comment:
This change is because on federated services, we no longer need to run that step